src/Node/Jwt.purs

```diff
@@ -87,7 +78,14 @@ instance showNumericDate :: Show NumericDate where
   show = unwrap >>> show
 
 instance decodeNumericDate :: Decode NumericDate where
-  decode = readInt >=> pure <<< NumericDate
+  decode =
+    readNumber >=> (_ * 1000.0) >>> Milliseconds >>> instant
+      >>> maybe
+          (throwError $ singleton $ ForeignError "Number value couldn't be turned into Instant")
+          (toDateTime >>> NumericDate >>> pure)
+
+instance encodeNumericDate :: Encode NumericDate where
+  encode = unwrap >>> fromDateTime >>> unInstant >>> unwrap >>> (_ / 1000.0) >>> floor >>> encode
```

Comment:
`/ 1000` and `floor` are necessary to keep the time in _seconds_, as per the rfc: https://tools.ietf.org/html/rfc7519#section-2